hudi-client/hudi-spark-client/pom.xml

```diff
@@ -174,6 +208,12 @@
       <artifactId>awaitility</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
What is this used for?  Does it have a compatible license?

Reply:
My assumption is you are referring to this dependency?
```
    <dependency>
      <groupId>com.thoughtworks.paranamer</groupId>
      <artifactId>paranamer</artifactId>
      <version>2.8</version>
      <scope>test</scope>
    </dependency>
```
this change was made by @xushiyan but i think we need this newer 2.8 for spark 3.2.1 if its the default profile. In master we have this https://github.com/apache/hudi/search?q=paranamer in the licenses we seem to be reffering to 2.7